### ISSUE TITLE
Support fingerprint reader 

### ISSUE THREAD

**Issue by mu22le** [mozilla-lockbox/lockbox-extension#272]
Entering the master password on a mobile device is time-consuming. This is more annoying on those platforms (such as Android) where the master password has to be re-entered each time the application is suspended (that is almost every time the user opens a different app).

Modern devices sport a fingerprint reader that could be used in lieu of passwords. It would be desirable for lockbox to support this feature as an alternative to the master password.

**Comment by devinreams** [comment]
This is absolutely high on our minds.

There's some foundational work we'd want to explore in Firefox itself to get access to the desktop's Touch ID API (only currently on MacBook Pro with Touch Bar). This is probably one of best related bugs to follow, too: https://bugzilla.mozilla.org/show_bug.cgi?id=1271851

And as this repository is dedicated to the Firefox extension I've also created an issue in our iOS repo as a placeholder (to acknowledge this is on our list to consider): https://github.com/mozilla-lockbox/lockbox-ios/issues/35

**Comment by cascella** [comment]
Great, thank you! Do you have any plans to port lockbox to android?

**Comment by devinreams** [comment]
Yes, we definitely want Lockbox to be available "everywhere" our target audience is, within reason.

Any details around priority, timing, implementation (eg: is Lockbox an extension on Firefox for Android, stand-alone app, both?), etc. are still being discussed. 😁
